sycl/source/detail/event_impl.cpp

```diff
@@ -407,6 +407,60 @@ event_impl::get_info<info::event::command_execution_status>() {
              : info::event_command_status::complete;
 }
 
+template <>
+typename info::platform::version::return_type
+event_impl::get_backend_info<info::platform::version>() const {
+  if (MContext->getBackend() != backend::opencl) {
```

Comment:
AFAIK  Event may have uninitialized context, context is initialized lazily for the event. For example, default-constructed event doesn't have context.

Reply:
Thanks for the reminder! I've changed this to use getContextImplPtr() which will initialize a default context instead of using MContext directly.